back-nest/src/app.module.ts

```diff
@@ -48,5 +37,6 @@ import { LoggerModule } from 'nestjs-pino';
     ScraperModule,
     RedisModule,
   ],
+  providers: [PrismaService], // Додано PrismaService до провайдерів
```

Comment:
Remove PrismaService from here. Also remove or translate comment to corresponding language